### DISCUSSION TITLE
How to use simple template funcion in ‘Raw String’？

### DISCUSSION THREAD

**Discussion by Awadabang** [cue-lang/cue#2788] [Q&A] [answered]
I'm migrating some deploy code from ansible to cue. When I write MySQL's deploy code, like: 
```
_config: #Config
apiVersion: "v1"
kind: "ConfigMap"
metadata: {
  name: "mysql"
}
data: {
  "my.cnf": """
    [client]
    port=\(#config.service.listenPort)
    socket=/run/mysqld/mysqld.sock
    [mysqld]
    {% if #config.mysql_skip_ssl is defined and #config.mysql_skip_ssl == 'yes' %}
    skip_ssl
    {% endif %}
  """
}
```
Is there a Best Practice to do something like this? or any other approach you would like to suggest! Thank you.

**Comment by mvdan** [comment, marked as answer]
You have interpolations in CUE like `"hello \(name)"`, but you also have templating which mimicks Go's text/template package, like `"hello {{ .name }}"`: https://pkg.go.dev/cuelang.org/go/pkg/text/template@master

**Reply by Awadabang** [reply]
Thanks!
